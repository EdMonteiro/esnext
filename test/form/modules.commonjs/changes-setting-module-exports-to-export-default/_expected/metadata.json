{
  "modules.commonjs": {
    "imports": [],
    "exports": [
      {
        "type": "default-export",
        "node": {
          "type": "ExpressionStatement",
          "expression": {
            "type": "AssignmentExpression",
            "operator": "=",
            "left": {
              "type": "MemberExpression",
              "object": {
                "type": "Identifier",
                "name": "module",
                "typeAnnotation": null
              },
              "property": {
                "type": "Identifier",
                "name": "exports",
                "typeAnnotation": null
              },
              "computed": false
            },
            "right": {
              "type": "Identifier",
              "name": "Foo",
              "typeAnnotation": null
            }
          }
        }
      }
    ],
    "directives": []
  }
}