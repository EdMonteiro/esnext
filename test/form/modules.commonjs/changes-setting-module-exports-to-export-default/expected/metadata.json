{
  "modules.commonjs": {
    "imports": [],
    "exports": [
      {
        "type": "default-export",
        "node": {
          "type": "ExpressionStatement",
          "expression": {
            "type": "AssignmentExpression",
            "operator": "=",
            "left": {
              "type": "MemberExpression",
              "computed": false,
              "object": {
                "type": "Identifier",
                "name": "module"
              },
              "property": {
                "type": "Identifier",
                "name": "exports"
              }
            },
            "right": {
              "type": "Identifier",
              "name": "Foo"
            }
          }
        }
      }
    ],
    "directives": []
  }
}